js/adapt-contrib-xapi.js

```diff
@@ -236,6 +253,15 @@ define([
         var val = this.getConfig('_' + key);
 
         if (val) {
+          // Note: xAPI wrapper requires a trailing slash and protocol to be present
+          if (key === 'endpoint') {
+            val = val.replace(/\/?$/, '/');
+
+            if (!/^https?:\/\//i.test(val)) {
+              val = 'http://' + val;
```

Comment:
Do you think it's worth logging this out as a warning?

Reply:
Yeah it would be useful to know